tools/bumper-bot/parser/parser.go

```diff
@@ -0,0 +1,48 @@
+package parser
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/pkg/errors"
+
+	"gopkg.in/yaml.v2"
+)
+
+type Component struct {
+	Name        string `yaml:"name"`
+	Url         string `yaml:"url"`
+	Commit      string `yaml:"commit"`
+	Release_tag string `yaml:"release_tag"`
+}
+
+type ComponentsConfig struct {
+	Components []Component
```

Comment:
We are missing `yaml:"components"` here

Reply:
IIUC, it doesn't require a yaml prefix since it only holds a struct, only leaf params (with values) get the "yaml" thing